DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/DependencyInjection/AutofacRhinoModule.cs

```diff
@@ -35,8 +35,8 @@ protected override void Load(ContainerBuilder builder)
     RegisterLoggerFactory(builder);
 
     // Register instances initialised by Rhino
-    builder.RegisterInstance<PlugIn>(SpeckleConnectorsRhino7Plugin.Instance);
-    builder.RegisterInstance<Command>(SpeckleConnectorsRhino7Command.Instance);
+    builder.RegisterInstance<PlugIn>(SpeckleConnectorsRhino7Plugin.Instance).SingleInstance();
```

Comment:
Did it fix something else?

Reply:
actually no, i should revert this change. i was looking into this https://spockle.atlassian.net/jira/software/c/projects/CNX/boards/5?selectedIssue=CNX-9509